src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/commands.py

```diff
@@ -57,7 +58,7 @@ def load_command_table(self, _):
         g.custom_command('create', 'aks_create', no_wait_param='no_wait')
         g.command('delete', 'delete', no_wait_param='raw', confirmation=True)
         g.custom_command('get-credentials', 'aks_get_credentials')
-        g.command('get-versions', 'get_upgrade_profile', table_transformer=aks_get_versions_table_format)
```

Comment:
If you like to ease the impact of the breaking change, you can still keep the old entry, but use the `deprecate_info` to tell users to use the new one. In the help, add `Deprecated` to the short summary of the old one. After one release, you can pull the old stuff out. Up to you though.
```python
 g.command('get-versions', 'get_upgrade_profile', table_transformer=aks_get_versions_table_format, deprecate_info="For accuracy, 'get-versions' has been renamed to 'get-upgrades'. Please start to use the new one")
```

Reply:
That's great 👍, I wasn't aware of the deprecation feature. I'll add that later today.

Reply:
@yugangw-msft I added a deprecation notice. Please let me know if there's anything else to change here. I'd really like to get this merged today for the next release (also #5466). 😄